cfme/tests/configure/test_access_control.py

```diff
@@ -85,6 +85,24 @@ def _check_item_visibility(item, user_restricted):
     return _check_item_visibility
 
 
+@pytest.fixture(params=['tag', 'tag_expression'])
+def tag_value(appliance, category, tag, request):
+    tag_type = request.param
+    if tag_type == 'tag':
+        tag_for_create = ([category.display_name, tag.display_name], True)
+        tag_for_update = ([category.display_name, tag.display_name], False)
+    else:
+        if appliance.version > '5.9':
```

Comment:
This should be `if appliance.version >= '5.9'` yes? or was it only active in a Z-stream?